sass/_programme.scss

```diff
@@ -0,0 +1,5 @@
+#programme-content {
+    #programme {
+        overflow-x: scroll;
+    }
+}
```

Comment:
something like this should fix the header pattern alignment (untested code!)
```suggestion
}

@media (min-width: 851px) {
  #programme-content {
    --pattern-height: 7;
  }
}
```

Reply:
Doesn't seem to fix, I'm not entirely sure what it does ._. ?
I think I'm going to leave it as is for now and see if we can fix it next week. I don't want to block the announcement on it

Reply:
#50